docs/as-thrift.md

```diff
@@ -54,25 +52,36 @@ function echo(context, req, head, body, callback) {
 }
 
 server.listen(4040, '127.0.0.1', onListening);
-
 function onListening() {
+    var peer = echoChannel.peers.add('127.0.0.1:4040');
+    peer.connect();
+    peer.waitForIdentified(onIdentified);
```

Comment:
Let's not do any waitForIdentified().

No need to `peer.connect()` use `channel.waitForIdentified({ host: host }, onIdentified)`